server/src/test/java/keywhiz/service/resources/automation/AutomationSecretResourceIntegrationTest.java

```diff
@@ -47,6 +54,26 @@
     mutualSslClient = TestClients.mutualSslClient();
   }
 
+  @Test
+  public void createsSecretWithOwner() throws Exception {
+    String secretName = UUID.randomUUID().toString();
+    String ownerName = UUID.randomUUID().toString();
+
+    createGroup(ownerName);
+
+    CreateSecretRequest request = new CreateSecretRequest(
+        secretName,
+        ownerName,
+        "description",
+        "content",
+        ImmutableMap.of(),
+        0);
+    createSecret(request);
+
+    AutomationSecretResponse details = getSecretByName(secretName);
+    assertEquals(ownerName, details.owner());
+  }
```

Comment:
This integration test verifies that we can create a secret with an owner via the deprecated automation API.